types/react-native/index.d.ts

```diff
@@ -1964,7 +1965,7 @@ type Falsy = undefined | null | false;
 interface RecursiveArray<T> extends Array<T | RecursiveArray<T>> {}
```

Comment:
Could extending `RecursiveArray` from `ReadonlyArray`, instead of `Array` solve this issue? Don't see how it can be written to.

Reply:
Unfortunately no :(